CONTRIBUTING.md

```diff
@@ -112,9 +113,33 @@ Varying levels of write access are made available via our project bot [Derek](ht
 
 If you have been added to the MAINTAINERS file in the root of an OpenFaaS repository then you can help us manage our community and contributions by issuing comments on Issues and Pull Requests. See [Derek](https://github.com/alexellis/derek) for available commands.
 
+**Who are the Core Contributors?**
+
+The Core Contributor group includes:
+
+- Alex Ellis (@alexellis)
+- Richard Gee (@rgee0)
+- Stefan Prodan (@stefanprodan)
+- Burton Rheutan (@burtonr)
+- LucasRoesler (@LucasRoesler)
+- Eric Stoekl (@ericstoekl)
+
+The Core Contributors have the ear of the project lead and help with strategy, project maintenance, community management and make a regular commitment of time to the project. Core Contributors attend all project meetings and calls.
+
+**Community/project meetings and calls**
+
+The community calls are held on Zoom on a regular basis with invitations sent out via email ahead of time.
+
+General format:
+
+- Project updates/briefing
+- Round-table intros/updates
+- Demos of features/new work from community
+- Q&A
+
 **Governance**
 
```

Comment:
Replace `lead` with `led`

Reply:
I believe it to be pronounced as `leed` as in Alex, not `lead` as in "be in the front"